test/e2e/atlas/profile_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Does it make sense to define a unit test instead of the e2e test (see https://github.com/mongodb/mongodb-atlas-cli/blob/master/internal/config/profile_test.go)? In general, e2e tests are more expensive to run than unit tests

Reply:
It does sense to have a unit test to test this.
The reason why I added a e2e test is because I wanted to make sure the global `--profile` flag is tested.

Do you think it would be better to only have unit tests? (and get rid of the e2e test?)

Reply:
I've added additional unit tests